packages/client/examples/light-browser-sync.md

```diff
@@ -0,0 +1,72 @@
+# Example: Light Browser Sync
+
+## Light sync 
+
+In this example, we will run two ethereumjs-clients. The first will be a full sync client that
+will connect to the rinkeby network and start downloading the blockchain. The second will be a
+light client that connects to the first client and syncs headers as they are downloaded.
+
+The first client will use RLPx to connect to the rinkeby network, but will also provide a libp2p
+listener. The second client will use libp2p to connect to the first client.
+
+Run the first client and start downloading blocks:
+
+```
+ethereumjs --syncmode full --lightserv true  --datadir first --network rinkeby --transports rlpx libp2p --multiaddrs /ip4/127.0.0.1/tcp/50505/ws
```

Comment:
Should we also include the bash instructions for when the client is run from the monorepo? i.e.:
```
npm run client:start  -- --syncmode full --lightserv true  --datadir first --network rinkeby --transports rlpx libp2p --multiaddrs /ip4/127.0.0.1/tcp/50505/ws
```
I think new users might be confused by the docs switching between the `ethereumjs` and `npm run client:start` commands

Reply:
yeah I agree, the new beginning of the README does explain this a bit, though for the examples i think we should stick to one or the other and not switch between.

Reply:
```suggestion
npm run client:start -- --syncmode full --lightserv true  --datadir first --network rinkeby --transports rlpx libp2p --multiaddrs /ip4/127.0.0.1/tcp/50505/ws
```